doc/register_file.rst

```diff
@@ -17,11 +17,40 @@ Note the flip-flop-based register file is significantly larger than the latch-ba
 Latch-Based Register File
 -------------------------
 
-The latch-based register file contains manually instantiated clock gating cells to keep the clock inactive when the latches are not written.
-
-It is assumed that there is a clock gating cell for the target technology that is wrapped in a module called ``prim_clock_gating`` and has the following ports:
+The latch-based register file uses manually instantiated clock gating cells to keep the clock inactive when the latches are not written.
+It is assumed that these technology-specific cells are wrapped in a module called ``prim_clock_gating`` that has the following ports:
```

Comment:
Can we move the requirements description of the clock gating cell to another place? This cell is used in ibex_core.sv as well as in ibex_register_file_latch.sv, so it might be hard to find this content. I'm thinking about a "how to get started with ibex" page somewhere, which explains necessary steps, such as "pick a clock gating primitive", "choose a register file", etc. (Doesn't need to go much beyond this for now, we can always extend later.)

Reply:
Great idea! I did that. We can always add more. For example, we could also add on how to use a future minimal SoC etc.